core/src/main/java/com/predic8/membrane/core/interceptor/MessageAnalyser.java

```diff
@@ -81,11 +85,16 @@ public String getShortDescription() {
 	}
 
 	@Override
-	public Outcome handleRequest(Exchange exc) throws Exception {
+	public Outcome handleRequest(Exchange exc) {
 
-		ExtractedData data = analyse(exc.getRequest());
+        ExtractedData data = null;
+        try {
+            data = analyse(exc.getRequest());
+        } catch (Exception e) {
+			log.error("Could not analyse request: {}",e.getMessage(), e);
```

Comment:
`e.getMessage()` is already printed, because `e` is passed to `log.error(...)` in the `Throwable` parameter.